cartofacade/capi.go

```diff
@@ -255,28 +255,28 @@ func (vc *Carto) addIMUReading(imu string, readings IMUReading, timestamp time.T
 	return nil
 }
 
-// GetPosition is a wrapper for viam_carto_get_position
-func (vc *Carto) getPosition() (GetPosition, error) {
+// position is a wrapper for viam_carto_get_position
```

Comment:
[q] are we choosing to keep the c++ code using get_position?

Reply:
I was wondering the same and decided to keep it as is, since it would be a lot of work to change the C++ code and it's not really that big of a deal to keep it as is.

Reply:
Lets file a JIRA ticket and talk about it as a team?

Reply:
https://viam.atlassian.net/browse/RSDK-4757